docs/topics/install.md

```diff
@@ -87,8 +87,100 @@ Brigade creates Persistent Volume Claims on the fly. By default it will create
 PVCs using the default persistent volume class. Changing your cluster's default
 storage class will change what Brigade creates.
 
-We recommend the following file system types for these distributions or platforms:
+The following volume backends have been tested:
 
 - Azure: AzureFile
 - Minikube: 9p (the default)
-- General: A storage type that supports ReadWriteMany.
+- Any: NFS
+
+The NFS solution is explained below.
+
+#### Using an NFS Server
+
+Brigade uses storage for caching and short-term file sharing. Because of this,
+it is often convenient to use storage backends that are optimized for short-term
+ephemeral storage.
+
+NFS (Network File System) is one protocol that works well for Brigade. You can
+use the [NFS Provisioner](https://github.com/IlyaSemenov/nfs-provisioner-chart)
+chart to easily install an NFS server.
+
```

Comment:
Note, one assumption is the host has nfs-common. Rare not to have it but we hit that once before.
If not already set up on your host, add nfs-common
`$ apt-get install nfs-common`

Reply:
That is _very_ good to know. Thanks!